application/v1/resources/views.py

```diff
@@ -75,29 +75,50 @@ def post(self):
             return {'message': 'email already exist'},400 
   
 
-        # #generate hash for user password
-        password = User.generate_hash(raw_password)
- 
+        # send validated user input to user model
+        new_user = User(
+            username=username ,
+            email=email,
+            password = User.generate_hash(password)
+     
+        )
 
- 
-        # # attempt sending user to user model
+        # attempt creating a new user in user model
         try:
-            result= User.create_user(username,email,password)
-
+            result = new_user.create_store_attendant()
             access_token = create_access_token(identity = username)
             refresh_token = create_refresh_token(identity = username)
-
             return {
                 'message': 'User was created succesfully',
                 'status': 'ok',
                 'access_token': access_token,
                 'refresh_token': refresh_token,
                 'user': result
-                }, 201
+                },201
 
         except Exception as e:
             print(e)
             return {'message': 'Something went wrong'}, 500
+
```

Comment:
remove the below code as it is the same as the code above

Reply:
I'll surely do so